README.md

```diff
@@ -59,7 +60,7 @@ Options can be used to tweak the functionality of the tests. They're used with t
 
 To run only a single feature file, run the command
 
-`mvn test -Dcucumber.options="src/test/resources/Features/filename.feature"`
+`mvn test -Dcucumber.options="src/test/resources/Features/UI OR API/filename.feature"`
```

Comment:
If this is a UI feature, this won't work without having the @Regression tag applied as well, right?

Would it make more sense to use a tag like @UI on those Before and After methods, and leave the @Regression tag alone? That way one could run all Regression tests with @Regression @UI, just the API ones with @Regression, and if you want to run a feature file that is related to UI tests you'd only need to add the @UI tag?

What I don't want to do is try to run two or three UI tests marked as @WIP, but be required to add the @Regression tag to do so, which would then mean I'd be running all of the Regression tests as well

Reply:
the whole point of having separate tags is because i dint want to invoke chrome browser every time i run API test. so we have to separate UI and API tags. we cant keep same tag for both.